dataland-backend/src/main/kotlin/org/dataland/datalandbackend/validator/PageRangeString.kt

```diff
@@ -0,0 +1,51 @@
+package org.dataland.datalandbackend.validator
+
+import jakarta.validation.Constraint
+import jakarta.validation.ConstraintValidator
+import jakarta.validation.ConstraintValidatorContext
+import jakarta.validation.Payload
+import kotlin.reflect.KClass
+
+/**
+ * Annotation for validating a page range as a string.
+ */
+@Target(AnnotationTarget.PROPERTY_GETTER)
+@Retention(AnnotationRetention.RUNTIME)
+@Constraint(validatedBy = [PageRangeValidator::class])
+annotation class PageRange(
+    val message: String = "Input validation failed: Invalid page range format.",
+    val groups: Array<KClass<*>> = [],
```

Comment:
The error message should mention the correct format with short examples.

Reply:
@tbfelder

Reply:
Please also add a small sentence describing the correct format (just like in the frontend).